### PR TITLE
fix(migration): The migration script will now migrate entries too

### LINKED ISSUE #102: Migration to native relations only works in development
### Describe the bug

[The part of the code that checks if there have been migrated from url alias to webtools](https://github.com/pluginpal/strapi-webtools/blob/6ed9ba55401ea62a3e0ec6a22bfd8272319b6adb/packages/core/server/admin-api/migrations/to-native-relation.ts#L7) does not work in production, since one would already have changed the pluginOptions during development.

### Expected behavior

I would expect the migration function to create links once.

### PR DESCRIPTION
**Pull request by Alexnortung**
### What does it do?

The migration script can now take care of migrating entries too, obsoleting some of the migration done during bootstrap.

### Why is it needed?

The migrations could not really run in production and would require two consecutive deployments, one where the old plugin option was still set, and a new one where the new one was set.

This PR works with just the migration script

### Related issue(s)/PR(s)

resolves #102

### CODE REVIEW

**Review by boazpoolman — CHANGES_REQUESTED** [review]
All-in-all it's approved.

Though you will need to write up a changeset for this to trigger a release.
For more info see https://github.com/changesets/changesets.

TLDR: Use `yarn add changeset`

**Comment by Alexnortung** [comment]
I also have a small change that I want to add to the script regarding to the inserts. If there is no entries to insert it will throw an error.

**Review by boazpoolman — APPROVED** [review]
LGTM
